Makefile

```diff
@@ -99,10 +99,9 @@ test_e2e:
 #   make test_e2e_node FOCUS=kubelet SKIP=container
 # Build and run tests.
 test_e2e_node:
-	hack/e2e-node-test.sh FOCUS=$(FOCUS) SKIP=$(SKIP)
+	hack/e2e-node-test.sh
```

Comment:
nit: All the options that can be passed to this makerule should be documented above. That way once we add `make help`, it is self-sufficient.